include/node_kernels/SDRSSTIDDESABLNodeKernel.h

```diff
@@ -0,0 +1,89 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2019 National Renewable Energy Laboratory.                  */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+
+#ifndef SDRSSTIDDESABLNODEKERNEL_H
+#define SDRSSTIDDESABLNODEKERNEL_H
+
+#include "node_kernels/NodeKernel.h"
+#include "FieldTypeDef.h"
+
+#include "stk_mesh/base/BulkData.hpp"
+#include "stk_mesh/base/Ngp.hpp"
+
+namespace sierra {
+namespace nalu {
+
+class Realm;
+
+class SDRSSTIDDESABLNodeKernel : public NGPNodeKernel<SDRSSTIDDESABLNodeKernel>
+{
+public:
+  SDRSSTIDDESABLNodeKernel(const stk::mesh::MetaData&);
+
+  KOKKOS_FORCEINLINE_FUNCTION
+  SDRSSTIDDESABLNodeKernel() = default;
```

Comment:
This will generate warnings on AppleClang because `const int nDim_` isn't initialized... so this constructor will be deleted.

Reply:
Ah good catch!

Reply:
@sayerhs do you have a preferred way of dealing with this?  I'd be fine with deleting the default constructor as long as we don't need it for the device. Seems like there are several of these kicking around based off my compiler warnings on the laptop, and I'd like to clean them up (in a separate PR) 😄

Reply:
@psakievich removing the default constructor is the right way to fix this (as we can't provide a value for `nDim_` without consulting Realm). We don't need the default constructor for the device as we are not creating a `Kokkos::View` of the `NodeKernel` but through a holder struct.

Reply:
:+1: thanks for clarifying that.  I wasn't sure how these kernels were being sent to device.